golang/cosmos/daemon/cmd/root.go

```diff
@@ -133,6 +135,15 @@ func initRootCmd(sender Sender, rootCmd *cobra.Command, encodingConfig params.En
 	}
 	server.AddCommands(rootCmd, gaia.DefaultNodeHome, ac.newApp, ac.appExport, addModuleInitFlags)
 
+	for _, command := range rootCmd.Commands() {
+		if command.Name() != "export" {
+			continue
+		}
+
+		extendCosmosExportCommand(command, sender != nil)
+		break
```

Comment:
You can save a few lines by inverting the test and using the implicit `continue` at the end of the loop body.

```suggestion
		if command.Name() == "export" {
			extendCosmosExportCommand(command, sender != nil)
		         break
		}
```

Reply:
Hmm, `sender != nil` is a bit opaque. Maybe it should be assigned to a named variable with an explanatory comment?